deployments/kai-scheduler/values.yaml

```diff
@@ -119,6 +119,18 @@ crdupgrader:
     pullPolicy: IfNotPresent
     # tag: ""  # Optional: Override global.tag or Chart.AppVersion
 
+topologyMigration:
+  # Enable migration of Kueue Topology CRs to KAI Topology CRs
+  # When enabled, a pre-install/pre-upgrade hook will:
+  # 1. Check if Kueue Topology CRD exists
+  # 2. Copy existing Kueue Topologies to KAI Topologies with the same spec
+  # The migrated topologies will have an annotation indicating they were migrated
+  enabled: true
+  image:
+    repository: bitnami/kubectl
```

Comment:
This will probably not pass security checks. You can use the same image as the crd upgrader https://github.com/NVIDIA/KAI-Scheduler/blob/main/deployments/crd-upgrader/Dockerfile